tests/ui/macros/genercs-in-path-with-prettry-hir.stdout

```diff
@@ -0,0 +1,15 @@
+#[prelude_import]
+use ::std::prelude::rust_2015::*;
+#[macro_use]
+extern crate std;
+//@ compile-flags: -Zunpretty=hir
+
+// issue#97006
+
+macro_rules! m { ($attr_path: path) => { #[$attr_path] fn f() {} } }
+#[
+
+inline]
```

Comment:
It will print `﻿inline<u8>` if not ignored, but I think the current output is acceptable since it stems from invalid code.